NEWS.md

```diff
@@ -36,6 +35,7 @@ Standard library changes
 
 #### Statistics
 
+* `mean` now calculates average for a function over dimensions ([#31576]).
```

Comment:
It actually did that before. The change here is allowing it to do so while also applying a mapped function.
```suggestion
* `mean` now accepts both a function argument and a `dims` keyword ([#31576]).
```